fluentui_others/src/main/res/drawable/contextual_command_bar_start_item_background.xml

```diff
@@ -28,4 +28,4 @@
             </item>
         </selector>
     </item>
-</ripple>
```

Comment:
Extra

Reply:
Do not see this in code. Not sure why showing like this

Reply:
I do not see this in the PR, can we resolve this?